Libraries/LibWeb/WebAssembly/Instance.cpp

```diff
@@ -59,7 +59,8 @@ void Instance::initialize(JS::Realm& realm)
             [&](Wasm::MemoryAddress const& address) {
                 Optional<GC::Ptr<Memory>> object = m_memory_instances.get(address);
                 if (!object.has_value()) {
-                    object = realm.create<Memory>(realm, address);
+                    // FIXME: consider an option to initialize shared memory.
```

Comment:
I don't quite understand this fixme, so maybe some more context/detail is necessary?

Reply:
Thanks for pointing this out! :)

I wasn't sure whether memory object should be always created with `Shared::No` or not. I've read the spec a bit more and it seems like shared memory is never expected in this case, hence the `FIXME` is removed.